CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   makes it possible to safely share `Future`s and `Stream`s between tests and
   their tear-downs.
 
+* Add support for IPv6.
```

Comment:
We should probably be clearer about the circumstances under which this actually caused problems. Maybe "Properly allocate ports when debugging Chrome and Dartium in an IPv6-only environment."